### PR TITLE
Policy agent first implementation

### PR DESCRIPTION
**Pull request by flashvoid**
https://paninetworks.kanbanize.com/ctrl_board/3/cards/299

### CODE REVIEW

**Comment by debedb** [comment]
Problem is that this shows all new code - any way to make a diff against the original python code?

**Comment by flashvoid** [comment]
I don't think it's possible but i can store diff in file and drop it here. Do we want it ?

**Comment by jbrendel** [comment]
I think that would be useful.

File: pkg/util/policy/agent/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Romana policy agent
+===================
+
+This agent is a tool to apply romana policies on all hosts.
+It accepts **Romana policy defenitions** which are not complete yet so current implementation is temporary.
```
Comment (debedb) [inline pkg/util/policy/agent/README.md]: defInitions

File: pkg/util/policy/agent/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+This agent intended to be contacted directly by platform adaptors or later by *policy manager*.
+In any case caller is responsible to convert policy from platform specific type to romana policy type.
+
+# Roadmap
+1. Update policy defenition.
+2. Maintain per tenant "policy vector" chains with list of policies applied to current tenant.
+3. Rewrite in Golang.
+3.1. Multitherading and locking
```
Comment (debedb) [inline pkg/util/policy/agent/README.md]: typo in Multithreading

**Comment by jbrendel** [comment]
Are we done with this review here?

**Comment by flashvoid** [comment]
Merged into `project/policy-vectors`.

**Pull request closed without merging**